### PR TITLE
LICENSE.txt: Use present instead of final year

### PR DESCRIPTION
**Pull request by bw-yuhogg**
Signed-off-by: Bob W. Hogg <rwhogg@linux.com>

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2009-2016 Homebrew contributors.
+Copyright 2009-2017 Homebrew contributors.
```
Comment (sjackman) [inline LICENSE.txt]: Should this be `present`?

Comment (bw-yuhogg) [reply LICENSE.txt]: Oops, yeah, I meant to do that.

Comment (maxim-belkin) [reply LICENSE.txt]: should we add
```
Copyright 20XX-present Linuxbrew contributors.
```
?

Comment (bw-yuhogg) [reply LICENSE.txt]: @maxim-belkin, interesting question. I've just been using "Homebrew" everywhere, even on code that originated entirely with us and not with upstream, to make it easier to copy code around between repos.

I don't personally believe it's necessary to add a Linuxbrew-specific line, but I'm not opposed to it either.

Comment (maxim-belkin) [reply LICENSE.txt]: Well, Linuxbrew is not Homebrew so, in my opinion, it makes perfect sense. But it's up to you and Shaun.

Comment (sjackman) [reply LICENSE.txt]: I'm personally fine with leaving it as Homebrew. I'd be willing to accept a PR to add
`Copyright 2009-present Homebrew and Linuxbrew contributors.`

**Pull request merged**